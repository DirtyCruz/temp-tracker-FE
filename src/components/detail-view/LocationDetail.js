import React from 'react';
import PropTypes from 'prop-types';

function LocationDetail({ temp }) {
  return (
    <h5>{temp}</h5>
  );
}

LocationDetail.propTypes = {
  temp: PropTypes.number.isRequired
};

export default LocationDetail;
