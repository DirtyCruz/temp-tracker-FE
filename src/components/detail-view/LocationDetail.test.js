import React from 'react';
import { shallow } from 'enzyme';
import LocationDetail from './LocationDetail';

describe('LocationDetail component', () => {
  it('renders LocationDetail', () => {
    const wrapper = shallow(<LocationDetail temp={6} />);
    expect(wrapper).toMatchSnapshot();
  });
});
