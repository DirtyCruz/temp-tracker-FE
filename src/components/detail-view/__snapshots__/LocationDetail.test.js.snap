// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationDetail component renders LocationDetail 1`] = `
<h5>
  6
</h5>
`;
